src/marqo/index.py

```diff
@@ -116,7 +116,7 @@ def refresh(self):
     def search(self, q: Union[str, dict], searchable_attributes: Optional[List[str]] = None,
                limit: int = 10, offset: int = 0, search_method: Union[SearchMethods.TENSOR, str] = SearchMethods.TENSOR,
                highlights=None, device: Optional[str] = None, filter_string: str = None,
-               show_highlights=True, reranker=None,
+               show_highlights=True, reranker=None, image_download_headers: Optional[Dict] = None,
```

Comment:
do we need to specify the type of the keys and values, like Dict[str, str]?

Reply:
I think we leave it quite flexible - because the header details can be different depending on customer use case